.scripts/continueTesting.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
*NO PR*

Dedicated script allowing to try a UI test multiple times without always starting from scratch